lib/authz/permissions_test.go

```diff
@@ -457,14 +455,20 @@ func newTestResources(t *testing.T) (*testClient, *services.LockWatcher, Authori
 	backend, err := memory.New(memory.Config{})
 	require.NoError(t, err)
 
+	t.Cleanup(func() {
+		require.NoError(t, backend.Close())
+	})
+
 	clusterConfig, err := local.NewClusterConfigurationService(backend)
 	require.NoError(t, err)
 	caSvc := local.NewCAService(backend)
 	accessSvc := local.NewAccessService(backend)
 	identitySvc := local.NewIdentityService(backend)
 	eventsSvc := local.NewEventsService(backend)
 
-	testClient := &testClient{
+	accessSvc.Close()
```

Comment:
This doesn't look right, did you mean to add it to t.Cleanup?

Reply:
Ahhh, this would explain the new testing error -- yes, thank you. Will correct!